### ISSUE TITLE
Two format string precision specifiers in one string breaks syntax highlighting

### ISSUE THREAD

**Issue by trvrmcs** [atom/language-python#305]
### Description

Syntax highlighting gets confused when given a string that contains two precision specifiers.

This works:
```python
def s():
    return f"{1000:,.2f} {2000}"

print(s())
```
This doesn't:
```python
def s():
    return f"{1000:,.2f} {2000:,.2f}"

print(s())
```

![syntax_highlighting_error](https://user-images.githubusercontent.com/45498500/58726030-ab1f1580-83ae-11e9-9dcc-24b86dd83db9.png)
  

I've simplified this example as much as possible; in longer files every line after the bad one fails highlight correctly.

### Versions
atom: 1.34.0
language-python: 0.51.8

**Comment by Arcanemagus** [comment]
> atom: 1.34.0

Atom v1.37.0 is the current stable release, you should try updating to that to see if this is resolved for you.

I'm unable to reproduce this in Atom 1.39.0-dev-1d9a4cafc, so if the fix isn't in Atom v1.37.0 it will be in a later release soon 😉:
![image](https://user-images.githubusercontent.com/427137/58735451-4fa15780-83af-11e9-97ee-0320c210276f.png)

Thanks for taking the time to file an issue and provide a minimal reproduction case!

**Comment by trvrmcs** [comment]
Thank you, upgrading to atom 1.37.0 fixed the problem.
